### PR TITLE
add new data in coding interview

### PR DESCRIPTION
**Pull request by harshcrop**

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -37,7 +37,10 @@ Hopefully by now, you have your own spreadsheet of different companies that you
 * [CSE 332](https://courses.cs.washington.edu/courses/cse332/17wi/): Data Structures and Parallelism (covers materials such as Priority Queues, Algorithm Analysis, Recurrences, Dictionaries, Sorting and Parallelism)
 * [Glassdoor](https://www.glassdoor.com/index.htm): Look at previous interview questions from companies right before you interview with them!
 * [Big-O Cheatsheet](http://bigocheatsheet.com/): Super useful :)
+* [LeetCode](https://leetcode.com/problemset/all/): Specially, if you apply for giant companies. The “medium” difficulty category is what you can expect in interviews.
```
Comment (mitalipalekar) [inline README.md:40]: I already have LeetCode in my repo!

File: README.md [hunk 2]
```diff
@@ -37,7 +37,10 @@ Hopefully by now, you have your own spreadsheet of different companies that you
...
+* [GeeksforGeeks](http://www.geeksforgeeks.org/): Great explanations for different algorithm questions. To me it’s a kind of organized algorithm question bank.
+* [Programming - InterviewBit](https://www.interviewbit.com/courses/programming/): Stepwise programing interview practice.
 
 ## Other Resources
 * [LSH Resources List](https://github.com/Ladies-Storm-Hackathons/Resources): Ladies Storm Hackathons is a diversity organization for women in tech.  This repository contains resources on interviewing, negotiating, attending hackthons and diversity focused student organizations.
-* [Hackathon Hackers](https://github.com/HackathonHackers): Comprehensive list of different HH subgroups as well as a list of other amazing opportunities
\ No newline at end of file
+* [Hackathon Hackers](https://github.com/HackathonHackers): Comprehensive list of different HH subgroups as well as a list of other amazing opportunities
```
Comment (mitalipalekar) [inline README.md:46]: I don't think anything changed here?

File: README.md [hunk 3]
```diff
@@ -37,7 +37,10 @@ Hopefully by now, you have your own spreadsheet of different companies that you
 * [CSE 332](https://courses.cs.washington.edu/courses/cse332/17wi/): Data Structures and Parallelism (covers materials such as Priority Queues, Algorithm Analysis, Recurrences, Dictionaries, Sorting and Parallelism)
 * [Glassdoor](https://www.glassdoor.com/index.htm): Look at previous interview questions from companies right before you interview with them!
 * [Big-O Cheatsheet](http://bigocheatsheet.com/): Super useful :)
+* [LeetCode](https://leetcode.com/problemset/all/): Specially, if you apply for giant companies. The “medium” difficulty category is what you can expect in interviews.
+* [GeeksforGeeks](http://www.geeksforgeeks.org/): Great explanations for different algorithm questions. To me it’s a kind of organized algorithm question bank.
```
Comment (mitalipalekar) [inline README.md:41]: Great resource :)

Comment (harshcrop) [reply README.md:41]: you can merge this pull to add this in the repo :)

Comment (mitalipalekar) [reply README.md:41]: @harshcrop Please fix my other comments and I will do so!

**Comment by mitalipalekar** [comment]
@harshcrop Closing this PR for now since you havent responded to my comments.  I'll add in your GeeksforGeeks suggestion in a separate addition myself. Thanks for the effort that you put into this!

**Pull request closed without merging**